addons/account_check_printing/models/account_payment_method.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class AccountPaymentMethod(models.Model):
+    _inherit = 'account.payment.method'
+
+    @api.model
+    def _get_payment_method_information(self):
+        res = super(AccountPaymentMethod, self)._get_payment_method_information()
```

Comment:
Fine tuning: Since python 3.0, it could be rewritten by `res = super()._get_payment_method_information()`